QuestPatcher/Views/PatchingView.axaml

```diff
@@ -13,17 +13,24 @@
     <DockPanel IsVisible="{Binding !IsPatchingInProgress}">
         <Panel Margin="20" DockPanel.Dock="Top">
             <StackPanel Margin="0 15 0 0" IsVisible="{Binding Config.ShowPatchingOptions}" Orientation="Horizontal" HorizontalAlignment="Left" VerticalAlignment="Top" Spacing="25">
-                <ToggleSwitch IsChecked="{Binding Config.PatchingPermissions.ExternalFiles}" IsEnabled="False">Allow External Files</ToggleSwitch>
-                <ToggleSwitch IsChecked="{Binding Config.PatchingPermissions.Debuggable}">Allow Debugging</ToggleSwitch>
-                <ToggleSwitch IsChecked="{Binding Config.PatchingPermissions.FlatScreenSupport}">Disable VR Requirement</ToggleSwitch>
                 <StackPanel Orientation="Vertical">
-                    <TextBlock Margin="0 25 0 0">Hand Tracking Type</TextBlock>
-                    <ComboBox Margin="20 10 20 20" SelectedIndex="{Binding Config.PatchingPermissions.HandTrackingType}">
-                        <ComboBoxItem>None</ComboBoxItem>
-                        <ComboBoxItem>V1</ComboBoxItem>
-                        <ComboBoxItem>V1 High Frequency</ComboBoxItem>
-                        <ComboBoxItem>V2</ComboBoxItem>
-                    </ComboBox>
+                    <StackPanel Orientation="Horizontal" HorizontalAlignment="Left" VerticalAlignment="Top" Spacing="25">
+                        <ToggleSwitch IsChecked="{Binding Config.PatchingPermissions.ExternalFiles}" IsEnabled="False">Allow External Files</ToggleSwitch>
+                        <ToggleSwitch IsChecked="{Binding Config.PatchingPermissions.Debuggable}">Allow Debugging</ToggleSwitch>
+                        <ToggleSwitch IsChecked="{Binding Config.PatchingPermissions.FlatScreenSupport}">Disable VR Requirement</ToggleSwitch>
+                        <StackPanel Orientation="Vertical">
+                            <TextBlock Margin="0 25 0 0">Hand Tracking Type</TextBlock>
+                            <ComboBox Margin="20 10 20 20" SelectedIndex="{Binding Config.PatchingPermissions.HandTrackingType}">
+                                <ComboBoxItem>None</ComboBoxItem>
+                                <ComboBoxItem>V1</ComboBoxItem>
+                                <ComboBoxItem>V1 High Frequency</ComboBoxItem>
+                                <ComboBoxItem>V2</ComboBoxItem>
+                            </ComboBox>
+                        </StackPanel>
+                    </StackPanel>
+                    <StackPanel Orientation="Horizontal" HorizontalAlignment="Left" VerticalAlignment="Top" Spacing="25">
```

Comment:
Why is the mic perms switch in its own panel?

Reply:
We ran out of horizontal space when questpatcher is at the minimum size, so I added an extra line for the extra toggle

Reply:
Ah ok that makes sense

Reply:
I messaged you on discord about that space last week